### PR TITLE
PkvStore Resource fix

### PR DESCRIPTION
**Pull request by Aultus-defora**
My attempt at fixing PkvStore not being a resource. Not sure if it is possible to add something smaller than bevy_ecs to dependencies. The error is `PkvStore: bevy::prelude::Resource is not satisfied` for `Res<PkvStore>`, `ResMut<PkvStore>`, ect.

### CODE REVIEW

**Comment by johanhelsing** [comment]
Thanks for the PR!

I wonder if it'd be possible to make bevy an optional dependency?

Something like:

```rust
#[derive(Debug)]
#[cfg_attr(feature = "bevy", derive(Resource))]
struct PkvStore {
```

Also, not sure what the practical difference is between importing bevy_ecs vs. importing `bevy` with `default-features = false` and then enable the features needed? I usually do the latter, but it would be good to know whether one approach has advantages over the other.

**Comment by Aultus-defora** [comment]
I am not sure what feature(s) is(are) enough to make #[derive(Resource)] work.

**Comment by Aultus-defora** [comment]
Now I use features to limit the Resource use, used cargo clippy to make linting good, also updated table — in 0.9 one has to at the very least create new struct with Resource to have PkvStore inside of a resource which is a hassle, this should solve that issue

File: README.md [hunk 1]
```diff
@@ -79,7 +79,8 @@ I intend to support the `main` branch of Bevy in the `bevy-main` branch.
 
 |bevy|bevy\_pkv|
 |---|---|
-|any|0.5, main|
+|0.8|0.6, main|
```
Comment (johanhelsing) [inline README.md:82]:
```suggestion
|0.9|0.6, main|
```

**Comment by johanhelsing** [comment]
Nice, this was exactly what I had in mind. Just a small error in the version table.

**Comment by Aultus-defora** [comment]
I merged the version table fix. Also confirmed that the `#[derive(Resource)]` works.

File: src/lib.rs [hunk 2]
```diff
@@ -1,4 +1,6 @@
 use serde::{de::DeserializeOwned, Serialize};
+#[cfg(feature = "bevy")] use bevy_ecs::prelude::Resource;
+
```
Comment (johanhelsing) [inline src/lib.rs:3]:
```suggestion
```

File: src/lib.rs [hunk 3]
```diff
@@ -31,6 +33,7 @@ pub use backend::{GetError, SetError};
 ///
 /// Automatically inserted when adding `PkvPlugin`
 #[derive(Debug)]
+#[cfg_attr(feature = "bevy", derive(Resource))]
```
Comment (johanhelsing) [inline src/lib.rs:36]:
```suggestion
#[cfg_attr(feature = "bevy", derive(bevy_ecs::prelude::Resource))]
```

**Review by johanhelsing — CHANGES_REQUESTED** [review]
Aside from fixing the linting error. with cfg attributes, I find it cleaner to just fully qualify types

**Review by johanhelsing — APPROVED** [review]

**Pull request merged**